### PR TITLE
setup finished

### PR DESCRIPTION
**Pull request by jasonmeeks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# Basic Calculator
+# [itp-w1] Highest Number Cubed
+
+Sometimes you can use programming to solve problems that would otherwise be difficult.
+For example, can you think of the highest number that, when cubed, is less than 12,000?
+Even using a calculator you would probably have to manually guess numbers until you figure it out. 
+Write a program that receives an upper limit and figures this out for you!
```
Comment (santiagobasulto) [inline README.md:6]: I really like this ^

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+All students will fork this repo and set up their cloud9 for this project individually with guidance from the mentor.
```
Comment (santiagobasulto) [inline README.md:8]: Also, good idea to include instructions here.

File: tests/test_main.py [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
-    def test_simple(self):
-        self.assertEqual(1 + 1, 2)
+    def test_three(self):
+        self.assertEqual(highest_number_cubed(30), 3)
+
+    def test_two(self):
+        self.assertEqual(highest_number_cubed(12), 2)
+
+    def test_one(self):
+        self.assertEqual(highest_number_cubed(3), 1)
```
Comment (santiagobasulto) [inline tests/test_main.py]: Can we add a few big number cases?

**Review by santiagobasulto — COMMENTED** [review]
+1, I'd add big numbers to make it more impressive

File: README.md (hunk 2 above)
Comment (jasonmeeks) [reply README.md:8]: I think the plan was to have the mentor walk them through it. Do you have a file already written with the instructions somewhere that I could copy paste it to here?

File: tests/test_main.py (hunk 3 above)
Comment (jasonmeeks) [reply tests/test_main.py]: absolutely :P

**Pull request merged**